package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "webpack-boilerplate",
+  "name": "to-do-list-pj",
   "version": "1.0.0",
-  "description": "",
+  "description": "> on this project I will be doing a to do list with webpack",
   "private": true,
   "scripts": {
```

Comment:
- [ ] [OPTIONAL] I Kindly suggest that you add steps in your README file on how to run your scripts locally. This will help users know how to run your project.

## EXAMPLE
```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/anasbassi/todo-list/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/anasbassi/todo-list.git
```
```
- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```
```
2. Start the development server

   ```sh
   npm start
   ```
```
3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.

## 🚀 Building and Running for Production

1. Generate a full static production build

   ```sh
   npm run build
   ```